### PR TITLE
p9: add xattr support

### PR DESCRIPTION
**Pull request by Lencerf**
This commit implements the new methods introduced in https://github.com/hugelgupf/p9/pull/66.

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -110,6 +110,8 @@ github.com/hexdigest/gowrap v1.1.7/go.mod h1:Z+nBFUDLa01iaNM+/jzoOA1JJ7sm51rnYFa
 github.com/hexdigest/gowrap v1.1.8/go.mod h1:H/JiFmQMp//tedlV8qt2xBdGzmne6bpbaSuiHmygnMw=
 github.com/hugelgupf/p9 v0.2.1-0.20230102222142-5cb68b4d03bb h1:rUzGqvx/TVHTtosnqlS8b4n7B4ClZ8emexHRethsWTQ=
 github.com/hugelgupf/p9 v0.2.1-0.20230102222142-5cb68b4d03bb/go.mod h1:1NBCX6Wq2cdaeDeagZuD5CygSuB5W5R5mnh/9e2DNPE=
+github.com/hugelgupf/p9 v0.2.1-0.20230803231326-7b9704081cf5 h1:uN6Q2LFhsP6mCEgmF12kAoWsuCHkjTO5zZ0gx2AKnd0=
```
Comment (brho) [inline go.sum:111]: i'm a little ignorant of go.mod stuff.  are these 2 new lines supposed to replace the previous p9 mod lines?

Comment (Lencerf) [reply go.sum:111]: I am not clear either. They are generated by `go get github.com/hugelgupf/p9@main`.

Comment (rminnich) [reply go.sum:111]: I wonder if go mod tidy would do a cleanup. The wonder and mystery of go.mod remain.

**Comment by rminnich** [comment]
ok don't be too mad at me, but I was hoping the union9p.go would break the build :-)
I just made more work for you, sorry!
But I was worried that a change would not be caught, and now I can worry less.

File: go.sum (hunk 1 above)
Comment (Lencerf) [reply go.sum:111]: `go mod tidy` works

**Comment by Lencerf** [comment]
> ok don't be too mad at me, but I was hoping the union9p.go would break the build :-) I just made more work for you, sorry! But I was worried that a change would not be caught, and now I can worry less.

Updated `union9p.go` in the new commit :)

**Review by rminnich — APPROVED** [review]

**Comment by rminnich** [comment]
here we go again.
rminnich@pop-os:~/go/src/github.com/u-root/cpu$ go test ./...
client/cpu9p.go:26:2: no required module provides package github.com/hugelgupf/p9/fsimpl/xattr; to add it:
	go get github.com/hugelgupf/p9/fsimpl/xattr
this is a fresh pull, checkout origin/main

the question being, why do our tests pass? 
rminnich@pop-os:~/go/src/github.com/u-root/cpu$ go version
go version go1.21rc3 linux/amd64

**Comment by Lencerf** [comment]
I do not have an arm machine. I tested on x86
```
[22:05:53][] desktop: ~/test
➜ git clone https://github.com/u-root/cpu
Cloning into 'cpu'...
remote: Enumerating objects: 3903, done.
remote: Counting objects: 100% (486/486), done.
remote: Compressing objects: 100% (222/222), done.
remote: Total 3903 (delta 261), reused 410 (delta 247), pack-reused 3417
Receiving objects: 100% (3903/3903), 3.60 MiB | 6.40 MiB/s, done.
Resolving deltas: 100% (2231/2231), done.
 
[22:06:06][] desktop: ~/test
➜ cd cpu
 
[22:06:09][] desktop: ~/test/cpu main
➜ go test ./...                          
?       github.com/u-root/cpu/cmds/cpu  [no test files]
?       github.com/u-root/cpu/cmds/decpu        [no test files]
ok      github.com/u-root/cpu/client    0.256s
ok      github.com/u-root/cpu/cmds/cpud 0.029s
ok      github.com/u-root/cpu/cmds/decpud       0.048s
ok      github.com/u-root/cpu/ds        12.053s
ok      github.com/u-root/cpu/mount     0.036s
ok      github.com/u-root/cpu/server    5.053s
ok      github.com/u-root/cpu/session   0.038s
 
[22:06:26][] desktop: ~/test/cpu main
➜ go version
go version go1.22-20230729-RC00 +457721cd52 X:fieldtrack,boringcrypto linux/amd64
```

**Comment by rminnich** [comment]
it was me making a mistake again :-)

**Pull request merged**